templates/dt_template/README.md

```diff
@@ -0,0 +1,58 @@
+# RHOS Deployed Topology %NEW_DT_NAME%
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | %TODAY%      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Compute nodes                                 | vm    | 3      |
+| Galera DB node, replicating                   | vm    | 3      |
+| Hardware security module                      | bm    | 1      |
+
+## Services, enabled features and configurations
+| Service                                     | configuration                   | Lock-in coverage?  |
+| ------------------------------------------- | ------------------------------- | ------------------ |
+| RabbitMQ                                    | default                         | Must have          |
+| OVN                                         | default                         | Must have          |
+| RabbitMQ                                    | default                         | Must have          |
+| Cinder                                      | Ceph/RGW backend                | optional           |
+| Manila                                      | Ceph/Ganesha+MDS Backend        | Must have          |
+| Glance                                      | NetApp, Fibre channel           | Must have          |
+| Keystone                                    | Special appliance [redacted]    | Interchangable     |
+| FIPS Mode                                   | default                         | Must have/standard |
+
```

Comment:
I would add an additional optional subsection ("### Additional configuration") to list "default" stuff (like TLSe and FIPS, just for reminder).

Reply:
Done